fix_checkpoint_history.py

```
@@ -0,0 +1,80 @@
+    history[target_csv].append({
+        "checkpoint_file": str(checkpoint_file),
+        "location_index": estimated_locations,
+        "timestamp": datetime.now().isoformat(),
+        "total_locations": 1607,
+        "measurements_count": line_count
+    })
+    
+    with open(history_file, 'w') as f:
+        json.dump(history, f, indent=2)
+    
+    print(f"Updated checkpoint history")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)